synapse/cores/lmdb.py

```diff
@@ -515,7 +640,9 @@ def _getSizeByProp(self, prop, valu=None, limit=None, mintime=None, maxtime=None
 
     def _getRowsByProp(self, prop, valu=None, limit=None, mintime=None, maxtime=None,
                        do_count_only=False):
-        indx = self.index_pt if valu is None else self.index_pvt
+        indx = self._dbs.get(PROP_VAL_INDEX)
```

Comment:
I had to write my own benchmarking tool: https://github.com/jnwatson/synapse/blob/master/scripts/benchmark_cortex.py

I'd be surprised if you noticed a difference on the hash times of short strings. It might take a fraction of a nanosecond difference. A dictionary lookup is in the single digit nanoseconds (at least on this box).